src/pages/Listing/AddListing/AddListing.container.js

```diff
@@ -133,7 +133,7 @@ const AddListing = () => {
           meetUpAddress: placeValue.formatted_address,
           meetUpInfo: new Date(year, month - 1, day, hours, minutes),
           name: itemName,
-          price: originalPrice / portionNumber,
+          price: portionPrice,
           qty: portionNumber,
         }).then(async (productResponse) => {
```

Comment:
enclose the portionPrice and portionNumber to Number() to make sure it is an integer.